www/_posts/2017-04-21-cordova-7.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Apache Cordova 7.0.0"
+categories: news
+tags: news releases
+---
+
+We are happy to announce that `Apache Cordova 7.0.0` has been released!
+
+Most notable changes include:
+
+* [cordova@7.0.0](https://www.npmjs.org/package/cordova)
+* [cordova-lib@7.0.0](https://www.npmjs.org/package/cordova-lib)
+* [plugman@1.5.0](https://www.npmjs.com/package/plugman)
+* [cordova-app-hello-world@3.12.0](https://www.npmjs.com/package/cordova-app-hello-world)
+* [cordova-common@2.0.3](https://www.npmjs.com/package/cordova-common)
+* [cordova-create@1.1.0](https://www.npmjs.com/package/cordova-create)
+* [cordova-fetch@1.1.0](https://www.npmjs.com/package/cordova-fetch)
+
+* If a `package.json` does not exist in your project, it will be auto-created for you when `cordova prepare` is called.
+* When adding a platform or plugin, the previous default behavior of not-saving has changed to saving. Now it will automatically save that platform or plugin to your `config.xml` and `package.json`. Details about platform and plugin versions are also automatically saved in `config.xml` and `package.json`. The `--save` flag is no longer required to save. Use `--nosave` to prevent saving to `config.xml` or `package.json`.
+* Fetch is now the default method for fetching platforms. Fetch uses your system `npm` to `npm install` modules into your project. The `--fetch` flag is no longer required. Use the `--nofetch` flag to revert to pre-`Cordova@7.0` behavior (`npm install` is not used to fetch modules).
+* After `cordova prepare` is run, `package.json` and `config.xml` should contain identical platforms and versions.  In case of conflicts, `package.json` is given precedence over `config.xml`. For example, suppose `package.json` contains `cordova-android@6.0.0` and `config.xml` contains `cordova-android@4.0.0`. After `cordova prepare` is run, `config.xml` and `package.json` will each contain only `cordova-android@6.0.0`.
+* Platforms and plugins are now required to have a `package.json` file.
+* We have added support for custom platforms. This will allow future custom platforms, and modified versions of existing platforms.
+
+	Example: 
+
+	cordova platform add custom-platform-name
```

Comment:
This (and the ones in the following paragraphs) aren't getting rendered as code blocks. You probably need to surround them with 3 backticks to force it.